pages/index.js

```diff
@@ -1,30 +1,73 @@
-import React from "react";
+import React, { useRef, useState} from 'react';
 
 import TodoInput from "../components/todo-input";
 import TodoList from "../components/todo-list";
 
 import styled from "styled-components";
 
 export default function Home() {
+
+  const [todos, setTodos] = useState([]);
+  const [index, setIndex] = useState(1);
+
+  const addTodo = ({content, date}) => {
+    if ((content === undefined) || (date === undefined)) {
+      alert('모든 항목을 입력해주세요!');
+      return;
+    }
+
+    else if (String(date).length != 8){
+      alert('날짜를 올바른 형식으로 입력해주세요!');
+      return;
+    }
+
+    const todo = {
+      id: index,
+      content,
+      date,
+      checked: false
+    }
+    setTodos(todos.concat(todo)); 
+    setIndex(index + 1);
+    alert('입력 완료!')
+  }
+
+  const todoRemove = (id) => {
+    setTodos(todos.filter(todo => todo.id !== id))
+  }
+
   return (
     <Wrapper>
+      <Header>리액트-투두</Header>
       <Contents>
-        <TodoInput />
-        <TodoList />
+        <TodoInput addTodo = {addTodo}/>
+        <Space/>
+        <TodoList todos={todos} todoRemove = {todoRemove}/>
       </Contents>
     </Wrapper>
   );
 }
 
 const Wrapper = styled.div`
-  height: 100vh;
+  min-height: 100vh;
+  background-color: rgb(155, 197, 195);
+  padding: 4rem 5rem;
 `;
 
 const Contents = styled.div`
-  border: solid 1px black;
   display: flex;
   flex-direction: row;
   justify-content: space-between;
-  padding: 10px 20px;
-  margin: 20px 10px;
+
+`;
+
+const Header = styled.p`
+  font-size: 3rem;
+  font-weight: 600;
+  padding: 0px;
+  margin: 0px 0px 3rem;
+`;
```

Comment:
```suggestion
const Header = styled.h1`
  font-size: 3rem;
  font-weight: 600;
  margin: 0px 0px 3rem;
`;
```
제목이니까 h1 태그 써주시면 좋을 것 같아요~ padding 0px는 여기에서도 필요 없어보입니다.